index.js

```diff
@@ -13,11 +13,17 @@ program
   .description(pkg.description);
 
 program
-  .command('top')
+  .command('top [articleCount]')
   .description('List Linda Ikeji Top Stories')
-  .option('-n, --number <int>", "specify number of stories')
-  .action(function(options){
-    var count = isNaN(parseInt(options) || parseInt(options) == 0) ? 20 : options;
+  .action(function(articleCount) {
+    var count = parseInt(articleCount) || 20;
+
+    // Prevent negative values
+    if (count < 0) {
+      count = 20;
+    }
```

Comment:
Yeah, just that 20 should be returned if the count is less than 0 like so: `count = (count < 0) ? 20 : count;`